forge-gui/res/cardsfolder/upcoming/inventory_management.txt

```diff
@@ -0,0 +1,8 @@
+Name:Inventory Management
+ManaCost:R W
+Types:Instant
+K:Split Second
+A:SP$ RepeatEach | UseImprinted$ True | RepeatCards$ Aura.YouCtrl,Equipment.YouCtrl | RepeatSubAbility$ DBAttach | SpellDescription$ For each Aura and Equipment you control, you may attach it to a creature you control.
```

Comment:
why extra params for Imprinted?

Reply:
Coudn't get this to work with remembered

Reply:
just tested, it will work